app/scripts/modules/core/src/widgets/spinners/Spinner.tsx

```diff
@@ -26,27 +46,25 @@ export class Spinner extends React.Component<ISpinnerProps> {
       rows.push(<div key={i} className="bar" />);
     }
     return rows;
-  }
+  };
 
-  public render(): React.ReactElement<Spinner> {
-    const { size, message, fullWidth } = this.props;
-    const messageClassNames = `message color-text-accent ${size === 'medium' ? 'heading-4' : 'heading-2'}`;
-
-    const messageNode = ['medium', 'large'].includes(size) && (
-      <div className={messageClassNames}>{message || 'Loading ...'}</div>
-    );
-
-    const bars = ['medium', 'large'].includes(size) ? (
-      <div className="bars">{this.getBarRows()}</div>
-    ) : (
-      this.getBarRows()
-    );
-
-    return (
-      <div className={classnames('load', size || 'small', { 'full-width': fullWidth })}>
-        {messageNode}
-        {bars}
-      </div>
-    );
-  }
-}
+  const messageClassNames = `message color-text-accent ${size === 'medium' ? 'heading-4' : 'heading-2'}`;
```

Comment:
I know that it was here before but it's kind of odd that only `medium` is using a smaller heading